packages/desktop-client/src/components/accounts/Header.tsx

```diff
@@ -5,6 +5,7 @@ import React, {
   type ReactNode,
   type ComponentProps,
 } from 'react';
+import { DialogTrigger } from 'react-aria-components';
```

Comment:
While I'm at it.. slightly cleaning up `Popover` usage. We no longer need to use refs/visibility variables. Instead we can just wrap it in `DialogTrigger`.

Reply:
Nice!